cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html

```diff
@@ -103,6 +103,12 @@ <h1 class="u-pullLeft">Alternative help</h1>
 
   <dir-pagination-controls on-page-change="pageChanged(newPageNumber)"></dir-pagination-controls>
 
+  <call-script>
```

Comment:
Does this not need a condition? I think it should only appear if they have failed assessment but in a scope category or in a category that is out of scope.